plugins/base/src/main/resources/dokka/styles/style.css

```
@@ -100,62 +76,421 @@ html ::-webkit-scrollbar-thumb {
     background: var(--color-scrollbar);
 }
 
+html, body {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    width: 100%;
+}
+/* /--- root styles --- */
+
+/* --- global tags styles --- */
+body, table {
+    background: var(--background-color);
+    font-family: var(--default-font-family);
+    font-style: normal;
+    font-weight: normal;
+    font-size: var(--default-font-size);
+    line-height: 24px;
```

One more minor comment.
I'd like to suggest replace 24px with the ratio (1.6), because it would work better in case of changing the font size. 
```suggestion
    line-height: 24px;
```